tensorflow-core/tensorflow-core-api/src/gen/annotations/org/tensorflow/op/Ops.java

```diff
@@ -8077,19 +8077,29 @@ public Ops withSubScope(String childScopeName) {
   }
 
   /**
-   * Returns an API that builds init operations.
-   * <p>Init operations will be initialized at session creation, will have their inputs (and control inputs) made init ops as well, and are never used as control dependencies.
-   * Additionally, this scope drops all of its control dependencies.  If an input can not be made an init op (i.e. a Placeholder), will error on op creation.
+   * Returns an API that builds init operations.  {@link #liftToInitScope(Operand)} will be called for all created operations.
+   * <p>
+   * Init operations will be initialized at session creation, will have their inputs (and control inputs) made init ops as well,
+   * and are ignored when used as control dependencies.
+   * Additionally, this scope ignores any control dependencies.
+   * <p>
+   * If an input can not be made an init op (i.e. a Placeholder), will throw an {@link IllegalStateException} on op creation.
+   * @see #liftToInitScope(Operand)
    */
   public Ops withInitScope() {
     return new Ops(scope.withInitScope());
   }
 
   /**
    * Make {@code op} an init operation, doing the same for all of it's inputs (and control inputs).
-   * <p>Init operations will be initialized at session creation, will have their inputs (and control inputs) made init ops as well, and are never used as control dependencies.
-   * Additionally, this scope drops all of its control dependencies.  If an input can not be made an init op (i.e. a Placeholder), will error on op creation.
-   * @throws IllegalArgumentException if the op or one of its inputs can't be made an init op.
+   * <p>
+   * Init operations will be initialized at session creation, will have their inputs (and control inputs) made init ops as well,
+   * and are ignored when used as control dependencies.
+   * Additionally, this scope ignores any control dependencies.
+   * <p>
+   * If an input can not be made an init op (i.e. a Placeholder), will throw an {@link IllegalStateException} on op creation.@see ExecutionEnvironment#registerInitOp(Operation)
```

Comment:
Put the `see` on a separate line, it'll be formatted like that in Javadoc anyway.